main.go

```diff
@@ -385,7 +385,7 @@ func main() {
 
 	err := app.Run(os.Args)
 	if err != nil {
-		log.Panicf("[bug] this code path shouldn't be arrived: helmfile is expected to exit from within the `cleanup` func in main.go: %v", err)
+		logger.Errorf("%v", err)
```

Comment:
Should we `os.Exit(1)` here? Otherwise this seems to return 0 even when there's an error.